crates/adapters/src/adhoc/table.rs

```diff
@@ -49,6 +50,7 @@ pub const fn datafusion_arrow_serde_config() -> &'static SqlSerdeConfig {
         date_format: DateFormat::String("%Y-%m-%d"),
         decimal_format: DecimalFormat::String,
         variant_format: VariantFormat::JsonString,
+        delimiter: CsvDelimiter::DEFAULT,
```

Comment:
It's odd that this needs to be part of `SqlSerdeConfig`, which is only supposed to contain properties related to deserializing individual data types, not how the parses handles the envelope. Is it possible to refactor this to not have `delimiter` in `SqlSerdeConfig`?

Reply:
OK, I'll figure it out. I didn't understand what was happening with SqlSerdeConfig.

Reply:
Fixed.